tests/server/tso/global_tso_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	. "github.com/pingcap/check"
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/kvproto/pkg/pdpb"
+	pdClient "github.com/tikv/pd/client"
```

Comment:
I think this test better should be placed in `tests/client.go`?

Reply:
`mismatch leader id` need `pd` enable local tso, so the test should be placed in `global_tso_test`.

Reply:
You can check `TestGlobalAndLocalTSO` in `tests/client/client_tests.go`. Enabling Local TSO is ok in there.

Reply:
updated.